organizing_hub/tasks.py

```diff
@@ -321,6 +321,20 @@ def send_event_promotion(event_promotion_id):
     if event_promotion.status != EventPromotionStatus.approved.value[0]:
         return sent_count
 
+    """If event is not approved, then do nothing"""
+    event = find_event_by_id_obfuscated(event_promotion.event_external_id)
+
+    if event.status != 'Approved':
```

Comment:
Need to add `None` handling if no event is returned.

Reply:
Also ideally would match on status id or key rather than human readable string. This was initially just meant for display content.